### PR TITLE
Support for --ansi flag

### PR DESCRIPTION
**Pull request by giftig**
I'd like to use ansi colour in my options but currently iterfzf won't pass the flag through. This will pass it through.

### CODE REVIEW

**Comment by giftig** [comment]
While I'm here, can I suggest exposing the ability to add extra arguments to the underlying command? You're not supporting every argument, but it'd be nice if I could unblock myself by providing `extra_args=['--ansi', '--color=light']` as raw input to work around it in the event that the feature isn't in yet or fzf releases a new feature.

It's very useful to see an fzf wrapper working so readily out of the box and handling installation of fzf itself though, so thanks for the project.

File: iterfzf/__init__.py [hunk 1]
```diff
@@ -45,6 +45,8 @@ def iterfzf(
         cmd.append('--exact')
     if multi:
         cmd.append('--multi')
+    if ansi:
+        cmd.append('--ansi')
```
Comment (giftig) [inline iterfzf/__init__.py:49]: I'll move this down actually, just noticed the other blocks are in the same order as the arguments.

File: README.rst [hunk 2]
```diff
@@ -189,6 +194,8 @@ Version 0.6.0.20.0
 
 To be released.  Bundles ``fzf`` 0.20.0.
 
+- Added support for the --ansi option
```
Comment (dahlia) [inline README.rst:197]:
```suggestion
- Added ``ansi`` option to ``iterfzf()`` function to support for the ``--ansi`` option.
```

File: README.rst [hunk 3]
```diff
@@ -141,6 +141,11 @@ The following is the full list of parameters.  Please pass them as
 
    Corresponds to ``--no-mouse`` option.
 
+``ansi``
+   ``True`` to enable processing ANSI color codes.  ``False`` by default.
+
+   Corresponds to ``--ansi`` option.
```
Comment (dahlia) [inline README.rst:147]:
```suggestion
   Corresponds to ``--ansi`` option.
   
   *New in version 0.6.0.*
```

**Comment by dahlia** [comment]
Ah, could you update *iterfzf/\_\_init\_\_.pyi* file as well?  It's for `typing` support.

**Pull request closed without merging**